prow/plugins.yaml

```diff
@@ -356,7 +356,11 @@ plugins:
 
   kubernetes/features:
   - approve
+  - blockade
   - milestone
+  - override
```

Comment:
Is this needed? There isn't testing on this repo.

Reply:
Good point. Removed.